package NGroundProto;

import "ground/protos/time_series_info.proto";
import "ground/protos/base.proto";

message TTimeSeries {
    optional TTimeSeriesInfo Info = 1;
    repeated TTimeSeriesData Data = 2;
}

message TTimeSeriesData {
    repeated double Values = 1;
}

message TTimeSeriesComplexData {
    repeated TComplex Values = 1;
}

message TTimeSeriesComplex {
    optional TTimeSeriesInfo Info = 1;
    repeated TTimeSeriesComplexData Data = 2;
}